macrosynergy/pnl/notional_positions.py

```diff
@@ -79,14 +169,10 @@ def _leverage_positions(
     df_wide: pd.DataFrame = qdf_to_ticker_df(df=df)
     sig_ident: str = f"_CSIG_{sname}"
 
-    _check_conts: List[str] = [f"{contx}{sig_ident}" for contx in fids]
-    if not set(_check_conts).issubset(set(df_wide.columns)):
-        raise ValueError(
-            f"Contract signals for all contracts not in dataframe. \n"
-            f"Missing: {set(_check_conts) - set(df_wide.columns)}"
-        )
+    _found_conts: List[str] = [f"{contx}{sig_ident}" for contx in fids]
```

Comment:
Why call it found? Should it just be `_contracts` or `_select_contracts`? To me found implies you might not have found all of them.

Reply:
Yes, this should be renamed. Earlier it was being used to actually check against a list of expected contracts, so I never renamed the variable